### DISCUSSION TITLE
Interact Zone Problem

### DISCUSSION THREAD

**Discussion by Lu1508** [Monika-After-Story/MonikaModDev#9390] [QA - code and content creation (technical)] [answered]
Hi! I'm working on a section where you can interact with Monika by clicking on her

However, I have this problem:

![Screenshot from 2022-07-23 18-10-27](https://user-images.githubusercontent.com/84401880/180666677-47a83c89-d930-49a0-8fb9-d185592f2967.png)
![Screenshot from 2022-07-23 18-10-55](https://user-images.githubusercontent.com/84401880/180666686-5e321292-2bcb-49cc-85bf-65d4775645cf.png)

How I can fix it?

**Comment by ThePotatoGuy** [comment, marked as answer]
are you talking about the menu in the 2nd image not having the touch button? If so, make sure you follow the pattern in the screen `hkb_overlay` in `zz_hotkey_buttons.rpy`

**Reply by Lu1508** [reply]
yup, it was that, I forgot to the "else" part

thank you so much!
